src/Features/Core/Portable/ExtractMethod/ExtractMethodMatrix.cs

```diff
@@ -73,6 +73,13 @@ static ExtractMethodMatrix()
                 {
                     key = new Key(dataFlowIn, /*dataFlowOut*/ true, alwaysAssigned, variableDeclared, readInside, writtenInside, readOutside, writtenOutside);
                 }
+
+                // interesting case in invalid code (bug #19136)
```

Comment:
Here I was taking the approach of a few lines above as well. Since the issue is also linked in the commit message (git blame), I could see this going either way.